runtime/autoload/provider/clipboard.vim

```diff
@@ -30,13 +30,11 @@ function! s:try_cmd(cmd, ...)
   return out
 endfunction
 
-let s:cache_enabled = 1
 if executable('pbcopy')
   let s:copy['+'] = 'pbcopy'
   let s:paste['+'] = 'pbpaste'
   let s:copy['*'] = s:copy['+']
   let s:paste['*'] = s:paste['+']
-  let s:cache_enabled = 0
```

Comment:
it is true that this safeguard isn't really needed, but it serves as documentation that pbcopy doesn't support caching. We should at least have a note here or something.